addons/l10n_din5008/models/chart_template.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    # Write paperformat and report template used on company
+    def _load(self, sale_tax_rate, purchase_tax_rate, company):
+        res = super(AccountChartTemplate, self)._load(sale_tax_rate, purchase_tax_rate, company)
+        if company.account_fiscal_country_id.code in ['AT', 'CH', 'DE']:
+            company.write({'external_report_layout_id': self.env.ref('l10n_din5008.external_layout_din5008').id,
+            'paperformat_id': self.env.ref('l10n_din5008.paperformat_euro_din').id})
+        return res
```

Comment:
This should be defined in the modules depending on this module (`l10n_{at,ch,de}`) with a
```python
if self == self.env.ref('l10n_xx.chart_template_reference'):
    company.write({
        'external_report_layout_id': self.env.ref('l10n_din5008.external_layout_din5008').id,
        'paperformat_id': self.env.ref('l10n_din5008.paperformat_euro_din').id
    })
```

Also, try to be consistent with the indentation. This was not really readable. Copy paste is bad :wink: